zap/src/main/dist/zap.sh

```diff
@@ -47,10 +47,10 @@ JAVA_VERSION=$(java -version 2>&1 | awk -F\" '/version/ { print $2 }')
 JAVA_MAJOR_VERSION=${JAVA_VERSION%%[.|-]*}
 JAVA_MINOR_VERSION=$(echo $JAVA_VERSION | awk -F\. '{ print $2 }')
 
-if [ $JAVA_MAJOR_VERSION -ge 11 ]; then
+if [ ${JAVA_MAJOR_VERSION:-0} -ge 17 ]; then
```

Comment:
We'll be changing the build to set it automatically.

Reply:
I will remove the commit